packages/alfa-aria/src/features/svg/animate-motion.ts

```diff
@@ -0,0 +1,10 @@
+import * as Roles from "../../roles";
+import { Feature, None } from "../../types";
+
+/**
+ * @see https://svgwg.org/specs/animations/#AnimateMotionElement
+ */
+export const AnimateMotion: Feature = {
+  element: "animatemotion",
```

Comment:
Element names in SVG are reported as defined in the SVG spec, so `<animateMotion>` would have a `localName` of `animateMotion` rather than `animatemotion`.